src/cur-mot/common/html_2014_2025.py

```diff
@@ -0,0 +1,552 @@
+                else:
+                    docD = add_to_docD(docD, div_d, current_div)
+                    current_div = "body"
+                    div_d = {"heading":{"type":"h1", "text":_.get_text().strip()}, "pars": []}
+
+
+            else:
+                if "pars" not in div_d:
+                    div_d['pars'] = []
+                if _.get_text().strip() is not None and _.get_text().strip() != '':
+                    t, look_for_notes = _ck_notes(_, look_for_notes)
+                    div_d['pars'].append({"p": t})
+
+        return docD, current_div, div_d, look_for_notes
+
+    ###########################
+    # OL UL OL UL OL UL OL UL #
+    ###########################
+    def  _handle_ol_ul(_, docD, div_d, current_div):
+    #    print("HERE!!!", current_div, div_d)
+        if "pars" not in div_d:
+            div_d['pars'] = []
+        div_d["pars"].append({"p": handle_list(_)})
+        return docD, current_div, div_d
+
+
+
+
+    # PUBLIC
+
+
+    docD = doc_D()
+    body_ck = True
+    soup = bs(html, "html.parser")
+    yrkande_pat = re.compile(r"-aw-sdt-title:'(Yrkande\s+[0-9]+)\'")
+
+    try:
+        soup = soup.find("div", class_="pconf")
+        assert soup is not None
+    except:
+        try:
+            soup = bs(f"<html>{html}</html>", "html.parser")
+        except:
+            fail_miserably("  No soup 1 -- parse_html -- I don't know what to do!")
+            with open("riksdagen-motions/no-soup.txt", "a+") as nosoup:
+                nosoup.write(f"{filename}\n")
+            return None
+        t = []
+        for _ in soup.descendants:
+            if type(_) == NavigableString:
+                t.append(_.strip())
+            else:
+                if _.string is not None:
+                    t.append(_.string.strip())
+        s = ' '.join([_.strip() for _ in t])
+        #print(s)
+        if "Motionen utgår" in s:
+            docD['body'] = [{"heading": {"type":"h1", "text": "Utgår"}, "pars": [{"p": s}]}]
+        elif "[~ konvertering pågår ~]" in s:
+            docD['body'] = [{"heading": {"type":"h1", "text": "Utgår"}, "pars": [{"p": s}]}]
+        else:
+            fail_miserably("  No soup 2 -- parse_html -- I don't know what to do!")
+            with open("riksdagen-motions/no-soup.txt", "a+") as nosoup:
+                nosoup.write(f"{filename}\n")
+            return None
+    else:
+        docD["header_title"] = soup.find("span", class_="sidhuvud_publikation").string.strip() + " " + \
+                            soup.find("span", class_="sidhuvud_beteckning").string.strip() + " " + \
+                            soup.find("span", class_="MotionarLista").string.strip() + " " + \
+                            soup.find("h1", recursive=False).string.strip()
+        section_1 = soup.find("div", class_="Section1")
+        current_div = None
+        look_for_notes = False
+        note_refs = []
+        div_d = {}
+        for i_, _ in enumerate(section_1):
```

Comment:
In loops, ```_``` indicates that the variable is not used, even if the iterator gives it to you.